manifests/base/kustomization.yaml

```diff
@@ -27,4 +27,4 @@ images:
   newTag: main
 - name: oauth-proxy
   newName: registry.redhat.io/openshift4/ose-oauth-proxy
-  newTag: v4.8
+  newTag: v4.10
```

Comment:
consider using digest instead of newTag. On the one hand, increases sameness, on the other hand, also makes the notation compatible with on-prem installs (digest) as well as with imagePullPolicy: IfNotPresent for using openshift node image cache

  digest: sha256:ab112105ac37352a2a4916a39d6736f5db6ab4c29bad4467de8d613e80e9bb33

Reply:
mmmm that's what @andrewballantyne commented above, we are planning to use digest in all our images, but we are going to do the migration all at once, we are going to keep the scope of this PR to just update the version of the oauth-proxy image.

Reply:
yeah, that is ok. Mainly saying that since you have imagePullPolicy: Always and the tag, you will have different images behind what is running together with the dashboard pod in terms of ose-oauth, depending on when the image was last pulled / the pod last started. You can never be sure that one running installation has that image with tag 4.10 and another has that image with v4.10 tag. tracking down an error might thus be harder to do. With digest notation, you can really pin down the image, plus ensure it is the same across all other odh components.

Reply:
I wanted to do this in layers, because I want us to take the time to get the right sha -- compare it with everyone, hopefully get a constant or something going.

This PR is to correct a lossy effect of us not paying attention to our "lowest supported OCP version"... Not the final solution. I onboarded last year and wasn't even close to looking at this version change, so this is a "catch up" effort.

I do get your point though, @shalberd -- it is much better to use sha digests everywhere because it is very specific and helps a lot. But I think it's important that we do these two things in two steps because conversations and ideas can slow down this basic upgrade.

I'm suggesting we continue the path with this PR as-is, and I'll log something in odh-manifests to see if we can unify across the board.

Reply:
Good idea. Regarding odh.manifests, I've already initiated a unifying PR https://github.com/opendatahub-io/odh-manifests/pull/869/files#diff-100131d24ded424b92afb1f01bc131586ed11798e6d1f2011d1633bafaf7c94d

Reply:
I made some comments here: https://github.com/opendatahub-io/odh-manifests/pull/869 to start things rolling that direction.